app/models/question.rb

```diff
@@ -7,15 +7,17 @@
 #  created_at                           :datetime         not null
 #  updated_at                           :datetime         not null
 #  question_type                        :string
+#  number                               :integer
```

Comment:
Can we move `question_type` and `number` above `created_at` since they're more important